examples/chip-tool/commands/tests/TestCommand.cpp

```diff
@@ -38,6 +52,10 @@ CHIP_ERROR TestCommand::Run(PersistentStorage & storage, NodeId localId, NodeId
     WaitForResponse(kWaitDurationInSeconds);
 
     mCommissioner.ServiceEventSignal();
+
+    // Give some time for all the pending messages to flush before shutting down
+    // Note: This is working around racy code in message queues during shutdown
```

Comment:
Please make sure there's an issue filed tracking this.  Especially if this is not already covered by #7297